src/converse-chatview.js

```diff
@@ -1018,6 +1026,25 @@ converse.plugins.add('converse-chatview', {
                 return this;
             },
 
+            async inject (ev) {
```

Comment:
`ev` is generally used to refer to an Event, looks like a string is passed in here?